### PR TITLE
Update docs for TransferTokens 'max' implementation

### PR DESCRIPTION
**Pull request by petitnau**
Documenting DeCash-Official/decash-on-algorand#50.

### CODE REVIEW

**Comment by petitnau** [comment]
Deprecation of CloseDelegatedWallet has already been documented, I don't think there's anything else to add.

File: docs/api/algorand/transfer-tokens.mdx [hunk 1]
```diff
@@ -4,11 +4,11 @@ Send DeCash from one wallet to another. This endpoint work for both regular and
...
-| `sender`    | `{address:string, amount:number}` | Object containing the `address` of the wallet that is sending the funds, and the `amount` that is being sent. Note that this account must have enough funds to cover both the `amount`, and the [fee](#TODO). |
-| `receiver`  | `string`                          | The address of the account that is receiving the funds. Note that, if this account is not a Delegated Wallet, then it must be opted into the token that is being sent. | 
+|     Name    |                   Type                   | Description |
+| ----------- | ---------------------------------------- | ----------- |
+| `assetId`   | `string`                                 | The ID of the DeCash asset that is being sent. |
+| `sender`    | `{address:string, amount:number\|"max"}` | Object containing the `address` of the wallet that is sending the funds, and the `amount` that is being sent. Note that this account must have enough funds to cover both the `amount`, and the [fee](#TODO). If `amount` is set to max, then all of the funds of asset `assetId` are sent to `receiver` (minus the required fee). |
```
Comment (nikitaeverywhere) [inline docs/api/algorand/transfer-tokens.mdx]: #TODO - we have already a page for that in /docs/api/fees

File: docs/api/algorand/transfer-tokens.mdx [hunk 2]
```diff
@@ -4,11 +4,11 @@ Send DeCash from one wallet to another. This endpoint work for both regular and
...
+| `receiver`  | `string`                                 | The address of the account that is receiving the funds. Note that, if this account is not a Delegated Wallet, then it must be opted into the token that is being sent. |
```
Comment (nikitaeverywhere) [inline docs/api/algorand/transfer-tokens.mdx:11]: Say an account has 0.01 USDD and they request a "max" transfer. What error would the user get?

**Review by nikitaeverywhere — APPROVED** [review]

File: docs/api/algorand/transfer-tokens.mdx (hunk 2 above)
Comment (petitnau) [reply docs/api/algorand/transfer-tokens.mdx:11]: They don't get any error as of right now in that case, an amount of 0 is transferred. If instead the fee was 0.02 and they had a balance of 0.01, then they would get a "not enough funds" error

Comment (petitnau) [reply docs/api/algorand/transfer-tokens.mdx:11]: They don't get any error as of right now in that case, an amount of 0 is transferred. If instead the fee was 0.02 and they had a balance of 0.01, then they would get a "not enough funds" error

**Pull request merged**